### PR TITLE
Refresh the network-ucopia-snmp documentation

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ucopia-snmp.md [hunk 1]
```diff
@@ -2,48 +2,503 @@
...
+2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.
+
+<Tabs groupId="sync">
+<TabItem value="Ha-Status" label="Ha-Status">
+
+| Macro                      | Description                                                                                                                                                  | Valeur par défaut | Obligatoire |
+|:---------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERCOUNTERS             | Only display some counters (regexp can be used). Example: --filter-counters='service-status'                                                                 | ha-status         |             |
+| DISPLAYOKCOUNTERS          |                                                                                                                                                              |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ucopia-snmp.md:232]: description?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ucopia-snmp.md [hunk 2]
```diff
@@ -2,48 +2,503 @@
...
+
+</TabItem>
+<TabItem value="Service-Status" label="Service-Status">
+
+| Macro                      | Description                                                                                                                                                  | Valeur par défaut      | Obligatoire |
+|:---------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------|:-----------:|
+| FILTERCOUNTERS             | Only display some counters (regexp can be used). Example: --filter-counters='service-status'                                                                 | service-status         |             |
+| DISPLAYOKCOUNTERS          |                                                                                                                                                              |                        |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ucopia-snmp.md:253]: description?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ucopia-snmp.md [hunk 3]
```diff
@@ -2,48 +2,503 @@
...
+
+</TabItem>
+<TabItem value="Temperature" label="Temperature">
+
+| Macro                      | Description                                                                                                                                                  | Valeur par défaut | Obligatoire |
+|:---------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERCOUNTERS             | Only display some counters (regexp can be used). Example: --filter-counters='service-status'                                                                 | temperature       |             |
+| DISPLAYOKCOUNTERS          |                                                                                                                                                              | temperature       |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ucopia-snmp.md:274]: description?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ucopia-snmp.md [hunk 4]
```diff
@@ -2,48 +2,503 @@
...
+
+</TabItem>
+<TabItem value="Users" label="Users">
+
+| Macro                      | Description                                                                                                                                                  | Valeur par défaut | Obligatoire |
+|:---------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERCOUNTERS             | Only display some counters (regexp can be used). Example: --filter-counters='service-status'                                                                 | users             |             |
+| DISPLAYOKCOUNTERS          |                                                                                                                                                              |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ucopia-snmp.md:295]: description?

File: pp/integrations/plugin-packs/procedures/network-ucopia-snmp.md [hunk 5]
```diff
@@ -2,48 +2,502 @@
...
+2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).
+
+<Tabs groupId="sync">
+<TabItem value="Ha-Status" label="Ha-Status">
+
+| Macro                      | Description                                                                                                                                                  | Default value     | Mandatory   |
+|:---------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERCOUNTERS             | Only display some counters (regexp can be used). Example: --filter-counters='service-status'                                                                 | ha-status         |             |
+| DISPLAYOKCOUNTERS          |                                                                                                                                                              |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-ucopia-snmp.md:233]: description?

File: pp/integrations/plugin-packs/procedures/network-ucopia-snmp.md [hunk 6]
```diff
@@ -2,48 +2,502 @@
...
+
+</TabItem>
+<TabItem value="Service-Status" label="Service-Status">
+
+| Macro                      | Description                                                                                                                                                  | Default value          | Mandatory   |
+|:---------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------|:-----------:|
+| FILTERCOUNTERS             | Only display some counters (regexp can be used). Example: --filter-counters='service-status'                                                                 | service-status         |             |
+| DISPLAYOKCOUNTERS          |                                                                                                                                                              |                        |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-ucopia-snmp.md:254]: description?

File: pp/integrations/plugin-packs/procedures/network-ucopia-snmp.md [hunk 7]
```diff
@@ -2,48 +2,502 @@
...
+
+</TabItem>
+<TabItem value="Temperature" label="Temperature">
+
+| Macro                      | Description                                                                                                                                                  | Default value     | Mandatory   |
+|:---------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERCOUNTERS             | Only display some counters (regexp can be used). Example: --filter-counters='service-status'                                                                 | temperature       |             |
+| DISPLAYOKCOUNTERS          |                                                                                                                                                              | temperature       |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-ucopia-snmp.md:275]: description?

File: pp/integrations/plugin-packs/procedures/network-ucopia-snmp.md [hunk 8]
```diff
@@ -2,48 +2,502 @@
...
+
+</TabItem>
+<TabItem value="Users" label="Users">
 
-  - Read-Only SNMP community
-  - IP Address of the equipment
+| Macro                      | Description                                                                                                                                                  | Default value     | Mandatory   |
+|:---------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERCOUNTERS             | Only display some counters (regexp can be used). Example: --filter-counters='service-status'                                                                 | users             |             |
+| DISPLAYOKCOUNTERS          |                                                                                                                                                              |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-ucopia-snmp.md:296]: description?

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**